### DISCUSSION TITLE
WebTransport server script

### DISCUSSION THREAD

**Discussion by pj8912** [aiortc/aioquic#382] [Q&A]
Is there a webtransport server script?
Or does [http3_server.py](https://github.com/aiortc/aioquic/blob/main/examples/http3_server.py) script can receive the streams(unidi&bidi) and datagrams ?
If yes, does it also work with WebTransport javascript client?

**Comment by aboba** [comment]
Here is the WebTransport server used in the WebTransport API Webplatform tests: 
https://github.com/web-platform-tests/wpt/tree/master/tools/webtransport

**Comment by pj8912** [comment]
Thank you @aboba

**Comment by pj8912** [comment]
So in `from aioquic.quic.connection import stream_is_unidirectional` 
is `stream_is_unidirectional` used to check that received stream is `unidirectional`? 
Is there anything for `bidirectional`??
